docs/locales/en/LC_MESSAGES/api/utils.po

```diff
@@ -613,7 +613,7 @@ msgstr ""
 msgid "The formatted string."
 msgstr ""
 
-#: 3dbe398f94684d7d81111c3a9d78ddee discord.utils.time_snowflake:1 of
```

Comment:
idk docs that good, but I feel like something should have been replaced, not removed

Reply:
it has been removed and replaced with generate_snowflake

Reply:
However idk how these files are generated, Lala knows

Reply:
The files are auntogenerated so I removed the changes and they will autoupdate when the automatic thing will be setup